app/routes/admin/email/components/RestrictedMailEditor.tsx

```diff
@@ -145,7 +145,6 @@ const RestrictedMailEditor = ({
         name="rawAddresses"
         placeholder="Enkelte eposter du ønsker å sende til"
         component={SelectInput.Field}
-        tags
```

Comment:
Was tags just completely dead code?

Reply:
No, it enables the user to enter new things into the selector. Under the hood it uses a different component from react select. So it needs to stay.

Reply:
I thought it was dead code because it wasn't much needed that one places I tested it 🙃 I fixed it now